packages/mobile/src/navigator/TopBarButton.v2.test.tsx

```diff
@@ -5,7 +5,7 @@ import TopBarButton from 'src/navigator/TopBarButton.v2'
 
 describe('TopBarButton', () => {
   it('displays children', () => {
-    const { queryByText } = render(<TopBarButton>label</TopBarButton>)
+    const { queryByText } = render(<TopBarButton title={'label'} />)
```

Comment:
hmm I think we should avoid changing interfaces

Reply:
I think it should be case-by-case. A title prop does make more sense to me than a child component

Reply:
I agree here with Rossy, plus it was introduced only in `v2`, so potentially will only affect our WIP branches. @aaronmgdr WDYT?